content/docs/visualization/promdash.md

```diff
@@ -129,6 +129,19 @@ interpolated version of the given format string. To reference specific label
 values in the format string, use double curly braces: `{{label-name}}`. For
 example: `{{host}} - cluster {{cluster}}`.
 
+### Legend format filters
```

Comment:
Hmm. Though labels in legend strings are not dashboard template variables. It's more like both cases use this feature, so maybe it should be separate somehow?